source/Willow-Bootstrap/BootstrapModalHeaderWebView.class.st

```diff
@@ -52,10 +55,9 @@ BootstrapModalHeaderWebView >> onCloseTrigger [
 BootstrapModalHeaderWebView >> renderContentOn: aCanvas [
 
 	aCanvas div
-		class: BootstrapCssStyles modalHeader;
 		with: [ dismissButtonOptional withContentDo: [ :dismissButton | aCanvas render: dismissButton ].
 			aCanvas heading
 				level4;
-				class: BootstrapCssStyles modalTitle;
-				with: title ]
+				with: title applying: [ :heading | heading addClass bootstrap modalTitle ] ]
```

Comment:
Since there is a nested use of components, it might improve readability to use `apply: ; with: [ apply: ; with: ]`